webapp/locales/en.json

```diff
@@ -159,6 +159,27 @@
       "labelBio": "About You",
       "success": "Your data was successfully updated!"
     },
+    "email": {
+      "validation": {
+        "same-email": "This is your current email address"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-30T09:12:20Z" title="Monday, September 30th 2019, 11:12:20 am +02:00">Sep 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -158,6 +158,28 @@
       "labelBio": "About You",
       "success": "Your data was successfully updated!"
     },
+    "email": {
+      "validation": {
+        "same-email": "Must be different from your current E-Mail address"
```


Different language, same idea: `This is your current e-mail address` :wink:

What you wrote sounds good in English but not very friendly in German – to my ears, at least. And if we decide to change it I think we should change it in both languages. I'll leave it up to you! :)